ldk/javascript/src/whisper/index.ts

```diff
@@ -230,19 +238,30 @@ export type ChildComponents =
   | Telephone
   | TextInput;
 
-export declare type CollapseBox = WhisperComponent<WhisperComponentType.CollapseBox> & {
+export type CollapseBox = WhisperComponent<WhisperComponentType.CollapseBox> & {
   children: Array<ChildComponents>;
   label?: string;
   open: boolean;
 };
 
-export declare type Box = WhisperComponent<WhisperComponentType.Box> & {
-  alignment: Alignment;
-  children: Array<Box | ChildComponents>;
+export type DeprecatedBox = WhisperComponent<WhisperComponentType.Box> & {
+  /**
+   * @deprecated - use justifyContent instead.
+   */
+  alignment: JustifyContent;
+  children: Array<ChildComponents>;
   direction: Direction;
 };
 
-export type Components = Box | ChildComponents | CollapseBox;
+export type Box = WhisperComponent<WhisperComponentType.Box> & {
+  justifyContent: JustifyContent;
+  children: Array<ChildComponents>;
+  direction: Direction;
+};
+
+export type Components = DeprecatedBox | Box | ChildComponents | CollapseBox;
+
+export type BoxChildComponents = ChildComponents | Box | DeprecatedBox;
```

Comment:
I'm a bit puzzled here. Box and DeprecatedBox already defined in the ChildComponents enum. Then why do we need to add them here?

Reply:
Oops! Good catch!

Reply:
Fixed!